### DISCUSSION TITLE
🇷🇺 Traffic separation: access to Youtube through 1 server, access to everything else through 2 servers. How to do it?

### DISCUSSION THREAD

**Discussion by goodboy34-tech** [Gozargah/Marzban#1192] [General]
اشتراک گذاری ترافیک: دسترسی به یوتیوب. چگونه این کار را انجام دهیم ؟ 🇷🇺

In Russia, YouTube is artificially slowed down, and this can cause problems when watching videos in normal quality. Bypassing the DPI (Deep Packet Inspection) can help to watch YouTube in better quality, but these tools are not available on smart TVs. Therefore, I am looking for a solution using Marzban. However, it is important to note that access to certain countries may be blocked in Russia, and I do not want to risk being arrested.
My question is: How can I implement traffic separation to ensure that I can watch YouTube without any issues while still being able to access other content?

در روسیه ، YouTube به طور مصنوعی کند می شود و این می تواند هنگام تماشای فیلم ها با کیفیت عادی مشکلاتی ایجاد کند. دور زدن DPI (بازرسی بسته عمیق) می تواند به تماشای YouTube با کیفیت بهتر کمک کند ، اما این ابزارها در تلویزیون های هوشمند موجود نیستند. بنابراین ، من به دنبال یک راه حل با استفاده از Marzban هستم. با این حال ، مهم است که توجه داشته باشید که دسترسی به برخی از کشورها ممکن است در روسیه مسدود شود ، و من نمی خواهم خطر دستگیری را داشته باشم.
سوال من این است: چگونه می توانم جداسازی ترافیک را پیاده سازی کنم تا اطمینان حاصل کنم که می توانم بدون هیچ مشکلی یوتیوب را تماشا کنم در 

حالی که هنوز هم می توانم به محتوای دیگر دسترسی داشته باشم ؟ 

В России работа YouTube искусственно замедлена, и это может вызвать проблемы при просмотре видео в нормальном качестве. Обход DPI (глубокой проверки пакетов) может помочь смотреть YouTube в лучшем качестве, но эти инструменты недоступны на смарт-телевизорах. Поэтому я ищу решение с помощью Marzban. Однако важно отметить, что в России доступ в некоторые страны может быть заблокирован, и я не хочу рисковать быть арестованным.
Мой вопрос заключается в следующем: как я могу реализовать разделение трафика, чтобы я мог смотреть YouTube без каких-либо проблем, сохраняя при этом доступ к другому контенту?

**Comment by localpurpose** [comment]
touch me in tg:@cestz
